File: templates/template.py
import sys

def main():
    input = sys.stdin.read
    print = sys.stdout.write

    # Code Here
    data = input().strip()
    number = int(data)
    print(f"{number}")


if __name__ == "__main__":
    if 'ONLINE_JUDGE' not in sys.argv:
        sys.stdin = open('input.txt', 'r')
        sys.stdout = open('output.txt', 'w')
    main()

File: templates/template.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <math.h>
#include <ctype.h>
#include <limits.h>
#include <float.h>
#include <time.h>
#include <stdbool.h>

#define OPTIMIZE_IO() {\
    setvbuf(stdout, NULL, _IONBF, 0); \
    setvbuf(stderr, NULL, _IONBF, 0); \
}

int main() {
#ifndef ONLINE_JUDGE
    freopen("input.txt", "r", stdin);
    freopen("output.txt", "w", stdout);
#endif
    OPTIMIZE_IO();

    // Your code here




    return 0;
}
